jwst/wavecorr/tests/test_wavecorr.py

```diff
@@ -171,13 +199,31 @@ def test_wavecorr_fs():
     dispersion = wavecorr.compute_dispersion(slit.meta.wcs, x, y)
     assert_allclose(dispersion[~np.isnan(dispersion)], 1e-8, atol=1.04e-8)
 
+    # Check that the slit wcs has been updated to provide corrected wavelengths
```

Comment:
As I see it this checks that `slit.meta.wcs` is consistent with `slit.wavelength`, regardless of whether it's updated or not. It's a good check but if you agree please change the comment.